whats_new/carts_vs_images.adoc

```diff
@@ -15,20 +15,22 @@ Cartridges in OpenShift v2 were the focal point for building applications.  Each
 However, with cartridges there was no clear distinction between the developer content and the cartridge content. For example, one limitation was that you did not have ownership of the home directory on each gear of your application. Another disadvantage was that cartridges were not the best distribution mechanism for large binaries. While you could use external dependencies from within cartridges, doing so would lose the benefits of encapsulation.
 
 == Dependencies
+Cartridge dependencies were defined with `Configure-Order` or `Requires` in the cartridge manifest.  OpenShift v3 uses a declarative model where pods bring themselves in line with a predefined state.  Explicit dependencies that are applied are done at runtime rather than just install time ordering.  For example, you might require another service is available before you start.  Such a dependency check is always applicable and not just when you create the two components.  Thus, pushing the dependency into a runtime dependency enables the system to stay healthy over time.
 
 == Collocation
 Whereas cartridges in OpenShift v2 could be collocated within gears, images are mapped 1:1 with containers. Containers use pods as their collocation mechanism.
 
 == Source Code
+In OpenShift v2, applications were required to have at least 1 web framework with 1 git repo.  With OpenShift v3, you can choose which images are built from source and that source can be located outside of OpenShift itself.  Because the source is disconnected from the images, the choice of image and source are distinct operations with source being optional.
 
 == Build
+In OpenShift v2 builds happened in place as part of you application gears.  This meant downtime for non scaled applications due to resource constraints.  In v3, builds happen in separate containers.
```

Comment:
Typo:

s/you application/your application/

Suggestion:

s/non scaled/non-scaled/